code/game/machinery/doors/door.dm

```diff
@@ -132,8 +132,12 @@
 
 /obj/machinery/door/CanAllowThrough(atom/movable/mover, turf/target)
 	. = ..()
-	if(istype(mover) && (mover.pass_flags & PASSGLASS))
-		return !opacity
+	if(istype(mover)) //yogs start
+		if(mover.pass_flags & PASSGLASS)
+			return !opacity
+		else if(mover.pass_flags & PASSDOOR)
+			return TRUE //yogs end
+	return .
```

Comment:
redundant

Reply:
?

Reply:
maybe I dunno

Reply:
Redundant as in remove it

A proc automatically returns ., no need to do it explicitly